DiffPlex/DiffBuilder/Model/DiffPaneModel.cs

```diff
@@ -5,6 +5,8 @@ namespace DiffPlex.DiffBuilder.Model
     public class DiffPaneModel
     {
         public List<DiffPiece> Lines { get; }
+        
+        public bool HasDifferences { get; internal set; } = false;
```

Comment:
false is default so no need to specify it I believe

Reply:
What about make this property nullable? It will be less confusing than `false` even by default.